### DISCUSSION TITLE
Issue with Column Definition Calculation Errors in QuestPDF Documentation

### DISCUSSION THREAD

**Discussion by ahmedbou2000** [QuestPDF/QuestPDF#760] [General]
Hi QuestPDF Team,

I've identified a calculation issue in the documentation. To assist in clarifying, I've attached an image highlighting the error. Looking forward to your insights.
( The Probleme is located at : https://www.questpdf.com/api-reference/table.html
Thanks,
![image](https://github.com/QuestPDF/QuestPDF/assets/109955655/840d2b33-23d4-4b81-950a-98c509e91eae)

**Comment by girlpunk** [comment]
Hi, could you share the code being used to generate this please? Thanks

**Comment by ahmedbou2000** [comment]
Hi , Yes , it's in the QuestPDF Documentation website .
[image: image.png]

**Comment by ahmedbou2000** [comment]
![table calc issue](https://github.com/QuestPDF/QuestPDF/assets/109955655/a2c73435-f34d-47bf-a6bb-d8d98500061c)

**Comment by ahmedbou2000** [comment]
the table is 300px size and the two **constantColumns** are 50px , and 100px sizes , so the 3th and the 4th columns are relative so they take from the remaining space ( 150px because the constant columns toke 50px + 100px) .
so the **RelativeColumn(2)** should takes : 150 * 2/5 ( 5 => 3 + 2 relative values ) .
the **RelativeColumn(3)** should takes : 150 * 3/5 .
am i wrong ?

**Comment by ahmedbou2000** [comment]
if you look at the image of the table , you will see that the 2nd and the 3rd columns are **not equals** , they said that they have the same size (100px) , **but it's not** .
the 2nd Column size is 100px 
the 3rd Column size is 60px as explained in my previous comment . 
tell me if im wrong ! 
thanks .

**Comment by MarcinZiabek** [comment]
Thank you for noticing and sharing this issue 😄  I am sorry for my delayed response. The example is fixed now: https://www.questpdf.com/api-reference/table.html#column-definitions

**Reply by ahmedbou2000** [reply]
Thank you MarcinZiabek ! 
Im enjoying QuestPDF
